app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,8 @@
 <resources>
     <string name="app_name">Fikt Exam</string>
-  <string name="title_home">Home</string>
-  <string name="title_dashboard">Dashboard</string>
-  <string name="title_notifications">Notifications</string>
+    <string name="title_home">Home</string>
+    <string name="title_dashboard">Dashboard</string>
+    <string name="title_notifications">Notifications</string>
+    <string name="StartScreenText">ТЕСТИРАЈТЕ ГО ВАШЕТО ЗНАЕЊЕ</string>
+    <string name="StartButton">ЗАПОЧНИ</string>
```

Comment:
This bring open question if we want MK to be default language or maybe we have other preferences?
For example we might have English and Macedonian.

But no matter what we choose the translation file should not be mixed.
In this file we have MK and EN, this is not acceptable.